docs/src/define-routes/cache.md

```diff
@@ -12,22 +12,23 @@ If a request is can be cached, Platform.sh builds a cache key from several reque
 
 When caching is on...
 
-* you can configure cache behaviour for different location blocks in your `.platform.app.yaml`;
+* you can configure cache behavior for different location blocks in your `.platform.app.yaml`;
 * the router will respect whatever cache headers are sent by the application;
 * cookies will bypass the cache;
 * responses with the `Cache-Control` header set to `Private`, `No-Cache`, or `No-Store` are not cached.
 
-You should _not_ use the Platform.sh HTTP cache if you're using Varnish or an external CDN.
+You should _not_ use the Platform.sh HTTP cache if you're using Varnish or an external CDN
+such as [Fastly](../domains/cdn/fastly.md) or [Cloudflare](../domains/cdn/cloudflare.md).
 Mixing cache services together most likely results in caches that are stale and can't be cleared.
 For more details, see [best practices on HTTP caching](../bestpractices/http-caching.md).
 
 ## Basic usage
 
-The HTTP cache is enabled by default, however you may wish to override this behaviour.
+The HTTP cache is enabled by default, however you may wish to override this behavior.
 
-To configure the HTTP cache, add a `cache` key to your route in `.platform/routes.yaml`. You may like to start with the defaults:
+To configure the HTTP cache, add a `cache` key to your route, as in the default:
```

Comment:
:nail_care: I'm not sure about the `in the default` phrasing. It seems slightly less clear than the original was, I think partly because of the `in`. But I wouldn't worry about it too much.